pom.xml

```diff
@@ -144,6 +143,16 @@
     </plugins>
   </build>
   <profiles>
+    <profile>
+      <id>default</id>
+      <activation>
+        <activeByDefault>true</activeByDefault>
+      </activation>
+      <modules>
+        <!-- This module only works when running it in JVM mode -->
```

Comment:
Maybe mention that it needs 10+ GB of RAM for native-image process?

It works if you have enough memory ...

Reply:
At the moment, It's not working, do you want me to create another pull request with the changes needed to make the 002 scenario work in Native?

Reply:
pls create PR to make it work

Reply:
https://github.com/quarkus-qe/beefy-scenarios/pull/118